docs/build/_partials/_oracles_contract_data.md

```diff
@@ -0,0 +1,28 @@
+import { AddToMetaMaskButton } from '@theme/AddToMetaMaskButton';
+import { Networks } from '@theme/constant';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId='network'>
+<TabItem value='iota' label='IOTA EVM'>
+
+|     Contract Type      |                                                                           Contract Address                                                                           |
+|:----------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|     Pyth contract      | [https://explorer.evm.iota.org/address/0x8D254a21b3C86D32F7179855531CE99164721933](https://explorer.evm.iota.org/address/0x8D254a21b3C86D32F7179855531CE99164721933) |
+|  Supra Pull Contract   | [https://explorer.evm.iota.org/address/0x2FA6DbFe4291136Cf272E1A3294362b6651e8517](https://explorer.evm.iota.org/address/0x2FA6DbFe4291136Cf272E1A3294362b6651e8517) |
+| Supra Storage Contract | [https://explorer.evm.iota.org/address/0xD02cc7a670047b6b012556A88e275c685d25e0c9](https://explorer.evm.iota.org/address/0xD02cc7a670047b6b012556A88e275c685d25e0c9) |
```

Comment:
```suggestion
```

Reply:
Pull and storage are the same contract, we can remove the mentioning of Storage and keep only Pull here